pkg/backend/local/progress.go

```diff
@@ -1073,12 +1066,25 @@ func getPreviewText(op deploy.StepOp) string {
 	return ""
 }
 
+func (display *ProgressDisplay) getStepOp(step engine.StepEventMetadata) deploy.StepOp {
+	op := step.Op
+
+	// In the progress-display, show all the steps for a replace as 'OpReplace'
+	if display.isTerminal {
```

Comment:
nit: `if display.isTerminal && (op == deploy.OpCreateReplacement || op == deploy.OpDeleteReplaced)`

Reply:
this is actually intentional on my part.  i tend to really dislike mixing/matching &&'s and ||'s in the same expression.  I find this easier to read myself.  Obviously for something small ike this it's not a big problem.  But as conditions tend to grow, i find this helps keeps hte ideas nicely separated.